bodhi/server/push.py

```diff
@@ -221,6 +221,9 @@ def _filter_releases(session, query, releases=None):
         _releases = session.query(Release).filter(or_(Release.state == ReleaseState.current,
                                                       Release.state == ReleaseState.pending))
 
+    # Filter only releases composed by Bodhi.
+    _releases = [r for r in _releases if r.composed_by_bodhi]
```

Comment:
Why not start the queries off with a query for releases that are composed by Bodhi, so we can let the database do the filtering? We could start a query before the ```if/else``` and then have the ```if/else``` add an additional ```.filter()``` call as appropriate.